packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -267,6 +267,11 @@ class ThemeData with Diagnosticable {
   ///  * [ThemeData.dark], which creates dark theme with a teal secondary [ColorScheme] color.
   ///  * [ColorScheme.fromSeed], which is used to create a [ColorScheme] from a seed color.
   factory ThemeData({
+    // For the sanity of the reader, make sure these properties are in the same
+    // order everywhere that they are separated by section comments (e.g.
```

Comment:
order everywhere that they are => order everywhere and that they are

Reply:
Well, that's not what I meant, though: I meant that everywhere they are listed and (already) separated by section comments, they should be in the same order.  I mean, yeah, they should also be separated by section comments everywhere that they're all listed too, but that's a different thing.

I'll change "everywhere" to be "in every place" if that helps.

Reply:
SGTM